presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -874,4 +879,62 @@ public HiveClientConfig setUseParquetColumnNames(boolean useParquetColumnNames)
         this.useParquetColumnNames = useParquetColumnNames;
         return this;
     }
+
+    public enum HiveMetastoreAuthenticationType
+    {
+        SIMPLE,
```

Comment:
I think we should name this NONE, and maybe rename `simpleHiveMetastoreAuthenticationModule` to `noHiveMetastoreAuthenticationModule`